packages/frontend/src/theme/foundations/colors.ts

```diff
@@ -2,6 +2,9 @@
 // size.
 // Imported from BbBear generated on Fri, 23 Jun 2023 09:14:33 GMT
 const bbBearColors = {
+  grey: {
+    '900': '#2C2E34',
+  },
```

Comment:
nit: is this used in another pr branch? cus i dont see this in the code haha

Reply:
wah good catch. I think i imported by accident.